aten/src/ATen/Context.cpp

```diff
@@ -411,6 +410,14 @@ void Context::setAllowBF16ReductionCuBLAS(bool b) {
   allow_bf16_reduction_cublas = b;
 }
 
+bool Context::useFP16AccumulationCuBLAS() const {
+  return use_fp16_accumulation_cublas;
```

Comment:
what is the difference between useFP16AccumulationCuBLAS and allowFP16ReductionCuBLAS?

Reply:
the reduction flag is not doing all accumulations in FP16 (less likely to have numerical issues like overflow) where as the accumulation flag will do all accumulations in FP16 (more likely to have overflow)